File: python3/ex023.py
num = int(input("Informe o numero: "))
u = num // 1 % 10
d = num // 10 % 10
c = num // 100 % 10
m = num // 1000 % 10


print(F"Analizando o numero {num}")
print(F"Unidade {u}")
print(F"Dezena {d}")
print(F"Centena {c}")
print(F"Milhar {m}")

File: javascript/ex023.js
console.clear();

const num = 1997;
u = num / 1 % 10
d = num / 10 % 10
c = num / 100 % 10
m = num / 1000 % 10


console.log(`Analizando o numero ${num}`);
console.log(`Unidade ${Math.floor(u)}`);
console.log(`Dezena ${Math.floor(d)}`);
console.log(`Centena ${Math.floor(c)}`);
console.log(`Milhar ${Math.floor(m)}`);